Solutions/Vectra/Hunting Queries/VectraSuspiciousNbOfSubDomains.yaml

```diff
@@ -0,0 +1,30 @@
+id: b8bacbda-3faa-49fb-b000-9334b2b51c8e
+name: Vectra AI - 
```

Comment:
Wondering if you missed giving a complete name to this.

Reply:
yes. I completed it and adjust the query to have a threshold.